### PR TITLE
Make tabs Hound-compliant

### PR DESCRIPTION
**Pull request by croaky**
It was a poor experience to add tabs to a project and then
have Hound provide lots of style feedback.

https://github.com/thoughtbot/hub/pull/411

This pull request attempts to address all that feedback.

### CODE REVIEW

**Comment by croaky** [comment]
@joshuaogle Can I get a review?

File: source/stylesheets/refills/_accordion-tabs.scss [hunk 1]
```diff
@@ -54,14 +54,14 @@
     color: $dark-gray;
     display: block;
     font-weight: bold;
-    padding: $base-spacing/2 $gutter/2;
+    padding: $base-spacing / 2 $gutter / 2;
```
Comment (joshuaogle) [inline source/stylesheets/refills/_accordion-tabs.scss]: Operations should have parentheses: `($base-spacing / 2) ($gutter / 2)`

**Comment by joshuaogle** [comment]
Small note, but otherwise this looks good :+1: 

There are a few lines in the JavaScript that are too long too, but that could be another PR

File: source/stylesheets/refills/_accordion-tabs-minimal.scss [hunk 2]
```diff
@@ -1,22 +1,22 @@
 .accordion-tabs-minimal {
-  $base-border-color: gainsboro !default;
-  $base-background-color: white !default;
+  $base-border-color: #dcdcdc !default;
+  $base-background-color: #ffffff !default;
```
Comment (tysongach) [inline source/stylesheets/refills/_accordion-tabs-minimal.scss]: I believe Hound (SCSS-Lint) will prefer this to be shortened to `#fff`

File: source/stylesheets/refills/_accordion-tabs.scss [hunk 3]
```diff
@@ -1,11 +1,11 @@
 .accordion-tabs {
-  $base-border-color: gainsboro !default;
+  $base-border-color: #dcdcdc !default;
   $base-border-radius: 3px !default;
-  $base-background-color: white !default;
+  $base-background-color: #ffffff !default;
```
Comment (tysongach) [inline source/stylesheets/refills/_accordion-tabs.scss]: I believe Hound (SCSS-Lint) will prefer this to be shortened to `#fff`

**Comment by tysongach** [comment]
Another minor note; otherwise :+1:

**Pull request merged**